README.md

```diff
@@ -58,3 +68,18 @@ public function calculate(DataProvider $dataProvider, float $multiplier)
 We have passed two arguments. One is `multiplier`. It is explicitly named. Such arguments passed as is. Another is 
 data provider. It is not named explicitly so injector finds matching parameter that has the same type.
 
+Creating an instance of an object of a given class behaves similar to `invoke()`:
+
+```php
+use Yiisoft\Injector\Injector;
+
+class StringFormatter
+{
+    public function __construct($string, \Yiisoft\I18n\MessageFormatterInterface $formatter)
+    {
+        // ...
+    }
+}
+
+$stringFormatter = (new Injector($container))->make(StringFormatter::class, ['string' => 'Hello World!']);
```

Comment:
Please add `StringFormatter` example that shows that `Injector` is working :)

Reply:
It is the example.

Reply:
I mean we need to show, that `$stringHelper->string` contains `Hello World!` and the second parameter of constructor is resolved